src/components/inline-color-picker/styles/editor.scss

```diff
@@ -1,4 +1,6 @@
 .components-coblocks-inline-color-picker {
+	opacity: 0.99;
```

Comment:
Interesting z-index and opacity quirk:
https://philipwalton.com/articles/what-no-one-told-you-about-z-index/